SR2019/2019-10-16-london-kickstart-deny.md

```diff
@@ -0,0 +1,14 @@
+---
+to: Student Robotics 2019, London Segment
+subject: The London kickstart is full
+note: This email is to be sent to those who are _not_ going to the London kickstart.
+---
+
+Hello!
+
+
+We regret to inform you that the London kickstart venue is full. There is space for you at the Southampton venue and we look forward to seeing you there instead.
+
+If you are unable to get to Southampton, then we can arrange another way to get you your kit.
+
+Please let Jake at [teams@] know which option you'd like, so we can make sure your kit gets to the right place.
```

Comment:
I suggest that we provide a full email address here rather than just "teams@". If that's just a placeholder and you plan to expand the address when sending, that's fine.

Reply:
Yes, see PR description.

Reply:
Is there any reason not to just include that address inline? We include it directly on the website, so it's already public.